docs/update-levelbuilder.md

```diff
@@ -3,14 +3,14 @@ This .md file should only contain information which is specific to Code.org engi
 
 # How to update level builder
 
-## To update levelbuilder to match staging:
+## To update levelbuilder to match test:
 
-1. On GitHub, open a pull request from `staging` into `levelbuilder`, link: [levelbuilder...staging](https://github.com/code-dot-org/code-dot-org/compare/levelbuilder...staging?expand=1)
+1. On GitHub, open a pull request from `test` into `levelbuilder`, link: [levelbuilder...test](https://github.com/code-dot-org/code-dot-org/compare/levelbuilder...test?expand=1)
   1. Or in your local repository:
```

Comment:
If there are two ways to do this then 9 should not be indented, it should be on the same level as 8.

Also, I'd emphasize that line 8 is the strongly preferred way to do it.